src/models/fire-model.ts

```diff
@@ -75,6 +82,14 @@ const FuelConstants: {[key in LandType]: Fuel} = {
   }
 };
 
+// values for each level of vegetation: Grass, Shrub, ForestSmallLitter, ForestLargeLitter
+export const moistureLookups: {[key in DroughtLevel]: number[]} = {
+  [DroughtLevel.NoDrought]: [0.1275, 0.255, 0.17, 0.2125],
+  [DroughtLevel.MildDrought]: [0.09, 0.18, 0.12, 0.15],
+  [DroughtLevel.MediumDrought]: [0.0525, 0.105, 0.07, 0.0875],
+  [DroughtLevel.SevereDrought]: [0.015, 0.03, 0.02, 0.025],
```

Comment:
Out of curiosity, where are these values coming from? Based on experiments or was that provided by someone from the project team? 

I guess Rothermal model specifies only moisture content for given vegetation type and there's no concept of drought level, right?

Reply:
Ah, I see the table in PT now.